views/index.mustache

```diff
@@ -1,12 +1,43 @@
-<html>
```

Comment:
So it seems like this whole file is a placeholder. We should probably just have the version that is currently in master here for now because it has the logout functionality.